src/SFA.DAS.ApprenticeCommitments.Database/Views/CommitmentStatementDashboardView.sql

```diff
@@ -0,0 +1,119 @@
+﻿CREATE VIEW dbo.CommitmentStatementDashboardView
+AS
+SELECT        Id, ApprenticeshipId, CommitmentsApprenticeshipId, CommitmentsApprovedOn, EmployerAccountLegalEntityId, EmployerName, TrainingProviderId, TrainingProviderName, CourseName, CourseLevel, PlannedStartDate, 
+                         CourseOption, PlannedEndDate, TrainingProviderCorrect, EmployerCorrect, RolesAndResponsibilitiesCorrect, ApprenticeshipDetailsCorrect, HowApprenticeshipDeliveredCorrect, ApprenticeshipConfirmed
+FROM            dbo.CommitmentStatement
+GO
+EXECUTE sp_addextendedproperty @name = N'MS_DiagramPaneCount', @value = 1, @level0type = N'SCHEMA', @level0name = N'dbo', @level1type = N'VIEW', @level1name = N'CommitmentStatementDashboardView';

```

Comment:
Does this dashboard need the CommitmentsApprovedOn field? Otherwise not sure how we distinguish the latest (if it's needed)

Reply:
Latest update should now include CommitmentsApprovedOn